test/controllers/class_controller_test.exs

```diff
@@ -253,4 +253,33 @@ defmodule CoursePlanner.ClassControllerTest do
     soft_deleted_course = Repo.get(Class, class.id)
     assert soft_deleted_course.deleted_at
   end
+
+  test "creates class and all attendance", %{conn: conn} do
+    course = insert(:course)
+    term1 = insert(:term, %{
+                            start_date: %Ecto.Date{day: 1, month: 1, year: 2017},
+                            end_date: %Ecto.Date{day: 1, month: 6, year: 2017},
+                            courses: [course]
+                           })
+
+    students =
+      [
+         insert(:user, %{role: "Student"}),
```

Comment:
what do you think of creating a `student_factory` in the factories file that already adds the role?